### PR TITLE
feat(liveness): add support for Kinesis

### PR DESCRIPTION
**Pull request by rohanmurthy**
Uses the [`listStreams` function](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/Kinesis.html#listStreams-property).

##### Checklist

- [x] `npm test` and `npm run lint` pass
- [x] tests are included
- [x] PR title follows [conventional commits guidelines](https://www.npmjs.com/package/@commitlint/config-conventional)

### CODE REVIEW

File: lib/aws-liveness.test.js [hunk 1]
```diff
@@ -40,14 +40,28 @@ describe('AWSLiveness', () => {
       const describeTablePromise = sinon.mock();
 
       describeTablePromise.promise = sinon.stub()
-        .resolves({ Table: { TableName: 'Foo' }});
+        .resolves({ Table: { TableName: 'Foo' } });
```
Comment (jeffpearce) [inline lib/aws-liveness.test.js]: Is this change intentional?

**Review by jeffpearce — APPROVED** [review]
Thanks for the PR!

**Review by JacopoDaeli — APPROVED** [review]
